tracee/tracee.go

```diff
@@ -388,7 +372,7 @@ func (t *Tracee) initBPF(ebpfObjectPath string) error {
 	//       5. Populate maps with values,
 	//       6. Attach probes,
 	//       7. Initialize perf buffers
-	t.bpfModule, err = bpf.NewModuleFromFile(ebpfObjectPath)
+	t.bpfModule, err = bpf.NewModuleFromFile(bpfObjectPath)
```

Comment:
If bpf t.bpfModule.BpfLoadObject() failed (line 406) - we can give the user a hint that he may need to recompile the code (this may happen if kernel version changed, and we use old bpf code binary)

Reply:
If we install to a tmp dir that is deleted after reboot - this is not an issue